tssc/step_implementers/validate_environment_configuration/configlint.py

```diff
@@ -0,0 +1,231 @@
+"""Step Implementer for the 'validate-environment-config' step for configlint.
+
+The Configlint step executes the config-lint against yml files for user-defined
+rules.   The inputs to this step include:
+
+  - Rules defined by the user in a specified file:
+
+    * Reference:  https://stelligent.github.io/config-lint/#/
+
+  - File path of yml files to lint
+
+    * Specify as a runtime argument, or
+    * Use results from previous step such as ConfilintFromArgocd
+
+Step Configuration
+------------------
+
+Step configuration key(s) for this step:
+
+| Key               | Required | Default                   | Description
+|-------------------|----------|---------------------------|-----------
+| `rules`           | False    | ./config_lint.rules       | File containing user-defined rules
+
+Expected Previous Step Results
+------------------------------
+
+Results expected from previous steps:
+
+| Step Name           |  Key                | Description
+|---------------------|---------------------|------------
+| `validate`          | `options.yml_path`  | Provides the path to the yml files
+| `environment-`      |                     | to be evaluated
+| `configuration`     |                     |
+
+Results
+-------
+
+Results output by this step:
+
+| Key                              | Description
+|----------------------------------|------------
+| `result`                         | A dictionary describing the output \
+                                     of this step
```

Comment:
Consider adding an `options` result key for the `rules` path to be shown when `rules` exists and `is not None` [Example](https://github.com/rhtconsulting/tssc-python-package/blob/979cca3d70f23046e4d6cb5d33ccdf40ca3db24d/tssc/step_implementers/unit_test/maven.py#L32)

Reply:
Added

Reply:
I see the documented definition and the example, but I don't see the implementation.

Reply:
I see it now :)